docker-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
         context: .
         dockerfile: Dockerfile
       environment:
```

Comment:
@RUFFY-369 Something I forgot earlier: Please also add the argument to activate / deactivate elastica here so its easily discoverable for people working with docker-compose

Reply:
Oh! Yeah that I'll add in the file

Reply:
Done in the latest commit